ecosystem/web-wallet/src/background.ts

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+import { AptosAccount, AptosClient, Types } from 'aptos'
+import { devnetNodeUrl } from './constants'
+import { MessageMethod } from './types'
+import { getAptosAccountState } from './utils/account'
+
+chrome.runtime.onMessageExternal.addListener(async function (request, _sender, sendResponse) {
+  const account = getAptosAccountState()
+  if (account === undefined) {
+    sendResponse({ error: 'No Accounts' })
+    return
+  }
+  switch (request.method) {
+    case MessageMethod.GET_ACCOUNT_ADDRESS:
+      getAccountAddress(account, sendResponse)
+    case MessageMethod.SIGN_TRANSACTION:
+      signTransaction(account, request.transaction, sendResponse)
+    default:
+      sendResponse({ error: 'Method not supported' })
```

Comment:
Is it possible to use assertNever so we can be notified at compile time if a new message type hasn't been implemented here?